README.md

```diff
@@ -2,115 +2,119 @@
+- `rdiv_wr`: Scale up a `Wad` value by one `Ray` scale and divide the scaled value by a `Ray` to return a `Wad`
+- `rdiv_ww`: Scale up a `Wad` value by one `Ray` scale and divide the scaled value by a `Wad` to return a `Ray`
+
+For multiplication, the prefixes `w` and `r` denote whether the product is divided (i.e. scaled down) by a `Wad` or `Ray` respectively. For division, the prefixes `w` and `r` denote whether the first operand is multiplied (i.e. scaled up) by a `Wad` or `Ray` before the division respectively.
+
+#### Zero and one values
 
-- `wmul(a, b)`, `rmul(a, b)`: Multiplication of `a` and `b`
-- `wsigned_div(a, b)`, `rsigned_div(a, b)`: Signed division of `a` by `b`
-- `wunsigned_div(a, b)`, `runsigned_div(a, b)`: Unsigned division of `a` by `b` with a bounds check that the result is within the range [-2<sup>125</sup>, 2<sup>125</sup>]
-- `wunsigned_div_unchecked(a, b)`, `runsigned_div_unchecked(a, b)`: Unsigned division of `a` by `b`
+Both `Wad` and `Ray` implement the `Zeroable` and `Oneable` traits.
+- `WadZeroable::zero()`/`RayZeroable::zero()`: Returns the zero value for `Wad` and `Ray` respectively
+- `is_zero()`: Returns true if the `Wad` or `Ray` value is zero
+- `is_non_zero()` Returns true if the `Wad` or `Ray` value is not zero
 
-### Aggregation
+#### Bound values
 
-These functions operate on `wad` only.
+Both `Wad` and `Ray` implement the `BoundedInt` trait.
+- `BoundedWad::max()`: Returns the maximum `Wad` value of 2<sup>128</sup> - 1
+- `BoundedWad::min()`: Returns the minimum `Wad` value of 0
+- `BoundedRay::max()`: Returns the maximum `Ray` value of 2<sup>128</sup> - 1
+- `BoundedRay::min()`: Returns the minimum `Ray` value of 0
 
-- `floor(n)` - Round a value down to the nearest `wad`
-- `ceil(n)` - Round a value up to the nearest `wad`
+### Comparisons
+
+Both `Wad` and `Ray` implement the `PartialEq` and `PartialOrd` traits.
+
+Comparison can be performed as follows:
+- `a == b`
+- `a != b`
+- `a > b`
+- `a >= b`
+- `a < b`
+- `a <= b`
+where both `a` and `b` are of the same type.
 
 These functions operate on any `ufelt` values in the range [0, 2<sup>125</sup>], including `wad` and `ray`.
 - `unsigned_min(a, b)` - Returns the smaller of two values
 - `unsigned_max(a, b)` - Returns the larger of two values
 
 ### Conversion
 
-- `to_uint(n)`: Convert a felt value to `Uint256`
-- `from_uint(n)`: Convert a `Uint256` value to felt with a bounds check that the result is within the range [-2<sup>125</sup>, 2<sup>125</sup>]
-- `to_wad(n)`: Multiply `n` by 10<sup>18</sup> with a bounds check that the result is within the range [-2<sup>125</sup>, 2<sup>125</sup>]
-- `wad_to_ray(n)`: Multiply `n` by 10<sup>9</sup> with a bounds check that the result is within the range [-2<sup>125</sup>, 2<sup>125</sup>]
-- `ray_to_wad(n)`: Divide `n` by 10<sup>9</sup>
-
-### Bounds Check
+The following conversions are supported by implementing the `Into` trait:
```

Comment:
"by implementing" sounds a little like the users still need to implement it. Maybe just saying "via the Into trait" or sth similar? Same below, with TryInto.

Reply:
I will change the phrasing across the README as well for clarity.